Changelog.md

```diff
@@ -2,6 +2,44 @@
 
 ## Gazebo Utils 2.0.0 (20XX-XX-XX)
 
+1. Fix CXX_STANDARD in core library
+    * [Pull request #77](https://github.com/gazebosim/gz-utils/pull/77)
```

Comment:
I would add this PR under the `Update GoogleTest` entry along with #63 since it is a follow-up to that PR

Reply:
Done in https://github.com/gazebosim/gz-utils/pull/79/commits/52b4a849f3e839bed7bdaa41d2547a344815ff5a